src/handlers/HeyHandler.ts

```diff
@@ -11,9 +13,9 @@ export default class HeyHandler implements Handler {
     regexps: HeyHandler.regexps,
   };
 
-  constructor(@inject(GroupMeService) private readonly groupMeService: GroupMeService) {}
+  constructor(@inject("MessagingService") private readonly messagingService: MessagingService) {}
```

Comment:
Why is this a string now?

Reply:
Because it is an interface. Classes have prototypes for getting injection tokens, but interfaces do not.

Reply:
TS being held back by JS is what I think it boils down to.